helm/metrics-server-app/templates/np.yaml

```diff
@@ -10,18 +10,17 @@ spec:
   podSelector:
     matchLabels:
       app: {{ .Values.name }}
+  policyTypes:
+  - Egress
+  - Ingress
   ingress:
   - ports:
     - port: {{ .Values.port }}
       protocol: TCP
   egress:
   - to:
+    {{ $privateSubnets := list "10.0.0.0/8" "172.16.0.0/12" "192.168.0.0/16" "100.64.0.0/10" -}}
```

Comment:
can't this value come from the installations folder like we do in other helm charts?

Reply:
It's not that easy, because this chart is installed in the tenant cluster and we don't have info about the installation there.
Adding the info requires a new release of cluster operator (where the info is not available as well).
I tried to do that all day, but gave up and stuck with the previous approach of hardcoding.